### PR TITLE
fix: transaction spawn status

### PR DESCRIPTION
**Pull request by maparr**

### CODE REVIEW

File: app/screens/node/Node.tsx [hunk 1]
```diff
@@ -386,16 +411,25 @@ const Node = ({ history, location }: Props) => {
...
+  const handleResumeSmeshing = () => {
+    if (isResumeButtonLocked) {
+      return;
+    }
+
+    setIsResumeButtonLocked(true);
+    dispatch(resumeSmeshing());
+    setTimeout(() => {
+      setIsResumeButtonLocked(false);
+    }, 1.2 * 60 * 1000);
```
Comment (brusherru) [inline app/screens/node/Node.tsx]: Can you describe this magic numbers? Mb add a comment

File: app/screens/node/Node.tsx [hunk 2]
```diff
@@ -412,27 +446,35 @@ const Node = ({ history, location }: Props) => {
...
-              onClick={handleResumeSmeshing}
-              text="RESUME POST DATA GENERATION"
-              img={playIcon}
-              isPrimary
-              width={280}
-              imgPosition="before"
-            />
-          )}
+          {!isPausedSmeshing &&
+            [PostSetupState.STATE_IN_PROGRESS].includes(postSetupState) && (
```
Comment (brusherru) [inline app/screens/node/Node.tsx]:
```suggestion
            postSetupState === PostSetupState.STATE_IN_PROGRESS && (
```
The same, but simpler, right? :)

File: app/screens/node/Node.tsx [hunk 3]
```diff
@@ -412,27 +446,35 @@ const Node = ({ history, location }: Props) => {
...
+                isPrimary={false}
+                width={280}
+                imgPosition="before"
+              />
+            )}
+          {isPausedSmeshing &&
+            [
+              PostSetupState.STATE_IN_PROGRESS,
+              PostSetupState.STATE_NOT_STARTED,
+            ].includes(postSetupState) && (
```
Comment (brusherru) [inline app/screens/node/Node.tsx]: In other words, it is not in completed state?

File: app/screens/node/Node.tsx [hunk 4]
```diff
@@ -475,7 +517,7 @@ const Node = ({ history, location }: Props) => {
 
   const handleSetupSmesher = () => {
     return eventsService
-      .switchApiProvider(LOCAL_NODE_API_URL, curNet)
+      .switchApiProvider(LOCAL_NODE_API_URL, currentGenesisId)
```
Comment (brusherru) [inline app/screens/node/Node.tsx]: 👍

**Review by brusherru — COMMENTED** [review]
Code looks well.
But I didn't test it yet. Will update asap

**Comment by maparr** [comment]
Hi @brusherru , changes for Smeshing screen were reverted, as a version with code we still have to check and the code will be in the history to pick it up in the future if needed.

**Review by brusherru — APPROVED** [review]
👌

**Pull request merged**